tests/Core/Microsoft.FluentUI.AspNetCore.Components.Tests.csproj

```diff
@@ -62,4 +62,11 @@
     
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Can you remove these new lines. Not required in this csproj.

Reply:
Sent #1769.